src/app/share/views/ShareView/ShareFolderView.tsx

```diff
@@ -182,6 +174,17 @@ export default function ShareFolderView(props: ShareViewProps): JSX.Element {
     return confirmationMessage; // WebKit, Chrome <34
   };
 
+  const getFolderSize = async (folderId) => {
+    try {
+      const folderSize = await shareService.getSharedFolderSize(folderId);
```

Comment:
[change the name of this, is not the folder size, is an object that contains the size](nit: rename this, as it is not the size of the folder, it is an object containing the size)

Reply:
Done